### PR TITLE
FPO-384-classification-table

### PR DESCRIPTION
**Pull request by Lee365**
### Jira link
https://transformuk.atlassian.net/browse/FPO-384

FPO-323

### What?

I have added script for new table

- [x] Added
- [x] Removed
- [x] Altered

### Why?

I am doing this because:

-

### AC

- Tested
- Shared with stakeholders
- Pair reviewed

### CODE REVIEW

File: src/migrations/20240820105236-create-classifications.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { type QueryInterface, type DataTypes } from 'sequelize'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up (queryInterface: QueryInterface, Sequelize: typeof DataTypes) {
+    await queryInterface.createTable('classifications', {
+      id: {
```
Comment (ashward) [inline src/migrations/20240820105236-create-classifications.ts:7]: Is it possible to make this a uuid field type?

Comment (Lee365) [reply src/migrations/20240820105236-create-classifications.ts:7]: Changed to uuid

File: src/models/classification.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  public id!: number
+  public descriptionId!: number
+  public description!: string
+  public commodityCode!: string
+
+  public readonly updatedAt!: Date
+  public readonly createdAt!: Date
+}
+
+Classification.init(
```
[the remark this answers is not in the document]
Comment (Lee365) [reply src/models/classification.ts:24]: This was generated by
npx sequelize db:migrate

[the remark this answers is not in the document]
Comment (ashward) [reply src/models/classification.ts:24]: Fair enough - must just be how it works!

**Review by ashward — APPROVED** [review]
lgtm

**Pull request merged**